portal/src/texts/profile/a11y.md

```diff
@@ -1,8 +1,10 @@
 ## Tilgjengelighet
 
-Universell utforming handler om at vi skal lage løsningene våre med tanke på at brukere kan ha ulik funksjonsevne, midlertidig eller varig. Når vi tenker på å utforme tilgjengelige løsninger, når vi alle målgruppene med én og samme løsning.
+Universell utforming handler om å lage løsninger som er lette å bruke for alle. Husk at våre brukere kan ha ulik funksjonsevne. Den kan være midlertidig eller varig nedsatt. Eksempler er synshemming, nedsatt finmotorikk, og nedsatte kognitive evner. Når vi utformer og utvikler løsninger som er tilgjengelige for disse brukerne, får vi løsninger som er bedre å bruke for alle målgrupper.
+
 En løsning er universelt utformet når den
-gir alle brukere samme muligheter, uansett funksjonsevne
-tilpasser seg brukeren uansett hvor i løsningen de befinner seg
 
-I løsninger som er rettet mot kunder, må Fremtinds produkter følge WCAG-standarden. Den spesifiserer hvordan ulike deler av løsningen skal være best mulig tilgjengelig for alle. Reglene gjelder ikke på samme måte for interne saksbehandlingsløsninger, men ved å følge gode prinsipper for design og tilgjengelighet, oppnår vi de beste løsningene uansett hvem som skal bruke dem.
+- gir alle brukere samme muligheter til å bruke den, uansett funksjonsevne
+- tilpasser seg brukeren, uansett hvor i løsningen de befinner seg
+
+Vi følger WCAG-standarden for alle løsninger som skal brukes av kunder. Det samme kravet gjelder ikke for interne saksbehandlingsløsninger, men vi tenker selvsagt på universell utforming og ønsker å lage de aller beste løsningene for alle, uansett hvem som skal bruke dem.
```

Comment:
My comment from earlier still applies in this edition. I think some text about how Fremtind is inclusive and how we'll make sure our internal applications is great for everyone, or just drop it here and keep those requirements internal. To me this sentence reads a bit negative.

Reply:
Please confer with @piofinn  and agree on how to phrase this, and I can proof it at the end.

Reply:
Updated with @piofinn suggestion and added a sentence about also describing accessibility for each profile element.